resources/js/Pages/Adminland/CompanyNews/Index.vue

```diff
@@ -34,20 +34,22 @@
       <div class="mw7 center br3 mb5 bg-white box restricted relative z-1">
         <div class="pa3 mt5">
           <h2 class="tc normal mb4">
-            {{ $t('account.company_news_title', { company: $page.props.auth.company.name}) }}
+            {{ $t('account.companylocalNews_title', { company: $page.props.auth.company.name}) }}
```

Comment:
No no 😀 you’ve also changed the translation strings, which won’t work, throughout the entire file and others too.

Reply:
Good catch! thanks